alica_engine/include/engine/syncmodule/SyncModule.h

```diff
@@ -39,15 +43,22 @@ class SyncModule
     void sendAcks(const std::vector<SyncData>& syncDataList) const;
     void synchronisationDone(const Synchronisation* st);
 
+    void reload(const YAML::Node& config);
```

Comment:
doesn't need to be public

Reply:
@triccyx please address this comment in any one of your later PRs. I am merging this so we can continue reviewing the other PRs.